roles/install-openjdk/tasks/main.yaml

```diff
@@ -5,3 +5,20 @@
       apt-get install openjdk-{{ java_version }}-jdk -y
     executable: /bin/bash
   environment: '{{ global_env }}'
+
+- name: Get host jvm path
+  shell: echo $(dirname $(dirname $(update-alternatives --list javac)))
+  no_log: yes
+  register: real_java_home
+
+- name: Set java env vars
+  set_fact:
+    java_env:
+      JAVA_HOME: "{{ real_java_home.stdout }}"
+      PATH: '{{ real_java_home.stdout }}/bin:{{ ansible_env.PATH }}'
+  no_log: yes
+
+- name: Merge java env vars into global env
+  set_fact:
+    global_env: '{{ global_env | combine(java_env) }}'
+  no_log: yes
```

Comment:
Please add a task to show `javac` path and version:
```
which javac
javac -version
```

Reply:
Done